test/bdd/fixtures/fabric/config/ledger/mychannel-blockchainhandler-config.yaml

```diff
@@ -6,3 +6,8 @@
 
 basePath: /sidetree/0.0.1/blockchain
 maxTransactionsInResponse: 10
+maxBlocksInResponse: 5
+
+authorization:
+  writeTokens:
```

Comment:
This needs to be readTokens - blockchain doesn't use writeTokens. If we don't want to restrict access to /blockchain then we don't need any authorization.

Reply:
ok

Reply:
done